modules/sms/scripts/send_sms.ts

```diff
@@ -0,0 +1,29 @@
+import { RuntimeError, ScriptContext } from "../module.gen.ts";
+import { Message } from "../utils/types.ts";
+import { TestProvider } from "../utils/providers/test.ts";
+import { TwilioProvider } from "../utils/providers/twilio.ts";
+import { SmsProvider } from "../utils/providers/index.ts";
+
+export interface Request {
+	to: string;
+	content: string;
+	useGSM7?: boolean;
```

Comment:
afaik we don't need to provide this, it's done by default: https://www.twilio.com/docs/glossary/what-is-gsm-7-character-encoding#how-twilio-encodes-your-messages

keep the api as minimal as possible